soccer/planning/IndependentMultiRobotPathPlanner.cpp

```diff
@@ -37,7 +37,7 @@ std::map<int, std::unique_ptr<Path>> IndependentMultiRobotPathPlanner::run(
 
     // Sorts descending so that higher priorities are first
     auto comparator = [&](int& lhs, int& rhs) {
-        return requests[lhs].priority > requests[rhs].priority;
+        return requests.at(lhs).priority > requests.at(rhs).priority;
```

Comment:
At checks bounds and throws an error. Also doesn't create a new object if it doesn't exist so it is const.